### DISCUSSION TITLE
Live search doesn't work in mobile chrome browser

### DISCUSSION THREAD

**Discussion by LF-Garcia** [livewire/livewire#3534] [1. Help] [answered]
Hi everyone,I am making a search input for live filtering, in chrome desktop works fine but in mobile it doesn't. please can you help me. 

views/usuarios/index.blade.php

@extends('layouts.adminlte')

@section('content')
   
    
    @livewire('show-user')

@endsection

resources\views\livewire\show-user.blade.php
```
<div>
    <div class="col-lg-4 mr-auto">
            <div class="input-group mb-3">
                <span class="input-group-text" id="basic-addon1"><i class="fa fa-search" aria-hidden="true"></i></span>
                <input wire:model="search" type="search" class="form-control" placeholder="Buscar" aria-describedby="basic-addon1">
            </div>
    </div>


    <hr>
    <div class="container pt-3">
        <div class="table-responsive">
            <table class="table" style="font-size: 12px;">
                <thead>
                <tr>
                    <th scope="col">#</th>
                    <th scope="col">Nombre</th>
                    <th scope="col">Apellido</th>
                    <th scope="col">Cedula</th>
                    <th scope="col">Rol</th>
                    <th scope="col">Correo</th>
                    <th scope="col">Accion</th>
                </tr>
                </thead>
                <tbody>
                    
                        @forelse ($usuarios as $usuariositem)
                        <tr>
                            <td >{{$usuariositem->id}}</td>
                            <td>{{$usuariositem->nombre}}</td>
                            <td>{{$usuariositem->apellido}}</td>
                            <td>{{$usuariositem->cedula}}</td>
                            <td>{{$usuariositem->rol}}</td>
                            <td>{{$usuariositem->email}}</td>
                            
                            
                            <td>
                                <div class="btn-group" role="group" aria-label="Basic example">
                                    <form action="{{ route('usuarios.edit', $usuariositem) }}" method="get" >
                                        @csrf
                                        <button type="submit" class="btn btn-secondary btn-xs">Editar</button>
                                    </form>
                                    
                                </div>
                            </td> 
                        </tr>
                        @empty
                            
                        @endforelse
                    
                        {{$usuarios->onEachSide(1)->links()}}
                </tbody>
            </table>
        </div>
            
    </div>
</div>
```

app\Http\Livewire\ShowUser.php

```
<?php

namespace App\Http\Livewire;

use Livewire\Component;


use App\Models\User;

class ShowUser extends Component
{
    

    public $search = ''; 


    public function render()
    {
        
        $usuarios = User::where('nombre', 'ILIKE', '%'.$this->search.'%')
                        ->orWhere('apellido', 'ILIKE', '%'.$this->search.'%')
                        ->orWhere('cedula', 'ILIKE', '%'.$this->search.'%')
                        ->orWhere('rol', 'ILIKE', '%'.$this->search.'%')
                        ->orWhere('email', 'ILIKE', '%'.$this->search.'%')
                        ->orderBy('id', 'ASC')
                        ->paginate(8);
        return view('livewire.show-user', compact('usuarios'));
    }
}
```

**Comment by joshhanley** [comment]
@LF-Garcia when you say it doesn't work? What do you mean? What happens? Can you provide some screenshots or error messages or something?

Thanks!

**Comment by LF-Garcia** [comment]
@joshhanley Thanks for you reply, I leave you a video [https://youtu.be/C157YLlcQnk](url)

**Reply by LF-Garcia** [reply]
In the video it's another project but I do the same like the code I posted in my question

**Comment by joshhanley** [comment]
@LF-Garcia thanks for the video!

Hmm I'm not sure what could be causing that, it seems like maybe Livewire isn't running properly when in mobile view? 
Maybe check that Livewire is running properly when in mobile layout?

Is it two different layouts or just the same layout that is responsive?

**Reply by LF-Garcia** [reply]
It's the same layout, am using adminlte. It's an admin template and it's responsive. I  not sure why it works properly in desktop and mobile doesn't. I'll keep trying. This was happen to you sometime ? @joshhanley

**Reply by joshhanley** [reply]
@LF-Garcia ah ok, yeah that is odd. Nah never had that happen to me.

In your phone emulator, can you pull up devtools for chrome by any chance and see if you can check that Livewire's scripts are loading and running?

**Comment by LF-Garcia** [comment, marked as answer]
@joshhanley great new's,

I solved my problem, let me explain:

1. am using laragon on my windows machine for dev porpouse, it use Virtual Host for every project, and redirect the trafic over http://proyectname.test url.

2. the @livewireScripts in the layout try to reach the source http://proyectname.test/livewire/livewire.js , in my computer its works, but laragon only redirects the trafic in my computer, not in my local network.

3. To test my app in mobile divices i was going to the url http://ip/proyectname/public/home of course all this whith my phone conected to my local network, and when the app loads it can't reach livewire scripts in http://ip/livewire/livewire.js.

Am publish the app in my server with public IP and livewire works fine. In the future am gona use a emulator in my dev machine to debugg my mobile enviroment.

**Reply by joshhanley** [reply]
@LF-Garcia ah nice! Good find 🙂
